operator/src/test/java/org/keycloak/operator/KeycloakDeploymentE2EIT.java

```diff
@@ -322,4 +322,16 @@ public void testInitialAdminUser() {
         }
     }
 
+    @Test
+    public void testCustomImage() {
```

Comment:
We should also add an explicit test for checking `--auto-build` is (not) present in args.

Reply:
done